### ISSUE TITLE
Remove SYSTEM:MODULE_CALL

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#732]
Update to this issue:

These MODULE_CALL is too limited and will instead be replaced with what is being implemented in #764 to resolve #665. Then MODULE_CALL has no meaning and should be removed. This is resolved in #767.

* 

There is a message pair, SYSTEM:MODULE_CALL/SYSTEM:MODULE_CALL_END that har sent out with level DEBUG, which show start and end of test case. There is, for some reason, a limitation so that they are only outputted if only one test case is tested.

~~They should always be outputted (with level debug).~~

No SYSTEM:MODULE_CALL/SYSTEM:MODULE_CALL_END are outputted:
```sh
zonemaster-cli ip.se --no-ipv6 --test DNSSEC --show_module --level debug --raw
zonemaster-cli ip.se --no-ipv6  --show_module --level debug --raw
```

SYSTEM:MODULE_CALL/SYSTEM:MODULE_CALL_END **are** outputted:
```sh
zonemaster-cli ip.se --no-ipv6 --test DNSSEC/dnssec13 --show_module --level debug --raw > dnssec13-raw
grep SYSTEM:MODULE_CALL dnssec13-raw 
   0.01 DEBUG     SYSTEM       SYSTEM:MODULE_CALL method=dnssec13; module=DNSSEC; version=v1.1.16
   3.80 DEBUG     SYSTEM       SYSTEM:MODULE_CALL_END method=dnssec13; module=DNSSEC
```

This issue relates to or duplicates #665

**Comment by vlevigneron** [comment]
In PR zonemaster/zonemaster-engine#764 discussion, it has been decided to remove theses MODULE_CALL messages.

**Comment by matsduf** [comment]
Resolved by #767
